code/modules/events/_event.dm

```diff
@@ -60,8 +60,8 @@
 
 	triggering = TRUE
 	if (alertadmins)
-		message_admins("Random Event triggering in 10 seconds: [name] ([typepath]) (<a href='?src=[REF(src)];cancel=1'>CANCEL</a>)")
-		sleep(100)
+		message_admins("Random Event triggering in 20 seconds: [name] ([typepath]) (<a href='?src=[REF(src)];cancel=1'>CANCEL</a>)")
+		sleep(200)
```

Comment:
replace that `200` with `20 SECONDS`

Reply:
I like how you care more about using `SECONDS` than you care about the use of `sleep()`